packages/fluentui/accessibility/src/behaviors/Datepicker/datepickerCalendarCellBehavior.ts

```diff
@@ -10,11 +10,15 @@ export const datepickerCalendarCellBehavior: Accessibility<DatepickerCalendarCel
   attributes: {
     root: {
       'aria-selected': props.selected || false,
```

Comment:
```suggestion
      'aria-selected': !!props.selected,
```

I know it's not part of these changes, but let's change it

Reply:
@mnajdova 
After the changes, I got the following error in pipeline. Should I then return the previous code?
```
datepickerCalendarCellBehavior.ts › Adds attribute 'aria-selected' based on the property 'selected' to 'root' slot.
ERR! 
ERR!     expect(received).toEqual(expected) // deep equality
ERR! 
ERR!     Expected: "value of property"
ERR!     Received: true
ERR! 
ERR!       108 |     property[propertyDependingOn] = propertyDependingOnValue;
ERR!       109 |     const expectedResult = parameters.behavior(property).attributes[elementWhereToBeAdded][attributeToBeAdded];
ERR!     > 110 |     expect(testHelper.convertToMatchingTypeIfApplicable(expectedResult)).toEqual(
ERR!           |                                                                          ^
ERR!       111 |       testHelper.convertToMatchingTypeIfApplicable(propertyDependingOnValue),
ERR!       112 |     );
ERR!       113 |   },
ERR! 
ERR!       at Object.testMethod (test/behaviors/testDefinitions.ts:110:74)
ERR!       at Object.test (test/behaviors/testHelper.tsx:60:24)
```

Reply:
Let's not tackle it now, it was not part of this PR anyway